src/components/Popover/Popover.module.css

```diff
@@ -0,0 +1,9 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+ # POPOVER CONTAINER
+\*------------------------------------*/
+
+.popover__panel {
+  overflow: visible !important;
```

Comment:
A tag controlled by HeadlessUI applies an inline style on this component, hence the `!important`.